router/core/graphql_handler.go

```diff
@@ -186,6 +187,22 @@ func logInternalErrorsFromReport(report *operationreport.Report, requestLogger *
 	}
 }
 
+func propagateSubgraphErrors(ctx *resolve.Context) {
```

Comment:
Can we call this method inside `writeRequestErrors` to not duplicate the context and span logic?